packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -2216,27 +2225,69 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/val
  
        ## Annotations for Grafana dashboard configmaps
        ##
-@@ -575,6 +936,19 @@
+@@ -574,7 +935,60 @@
+   ## Passed to grafana subchart and used by servicemonitor below
    ##
    service:
-     portName: service
+-    portName: service
++    portName: nginx-http
 +    ## Port for Grafana Service to listen on
 +    ##
 +    port: 80
 +    ## To be used with a proxy extraContainer port
 +    ##
-+    targetPort: 3000
++    targetPort: 8080
```

Comment:
Modified to point to the nginx proxy port instead of the Grafana container port.